The AllConfigurations build is failing:

```
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\Microsoft.XmlSerializer.Generator\Debug\netstandard2.0\dotnet-Microsoft.XmlSerializer.Generator.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\Microsoft.XmlSerializer.Generator\src\Microsoft.XmlSerializer.Generator.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\System.Runtime.Serialization.Schema\Debug\net7.0\System.Runtime.Serialization.Schema.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\System.Runtime.Serialization.Schema\src\System.Runtime.Serialization.Schema.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\Microsoft.Bcl.AsyncInterfaces\Debug\net462\Microsoft.Bcl.AsyncInterfaces.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\Microsoft.Bcl.AsyncInterfaces\src\Microsoft.Bcl.AsyncInterfaces.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\Microsoft.Bcl.Cryptography\Debug\net462\Microsoft.Bcl.Cryptography.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\Microsoft.Bcl.Cryptography\src\Microsoft.Bcl.Cryptography.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\System.Numerics.Tensors\Debug\net462\System.Numerics.Tensors.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\System.Numerics.Tensors\src\System.Numerics.Tensors.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\System.Threading.RateLimiting\Debug\net462\System.Threading.RateLimiting.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\System.Threading.RateLimiting\src\System.Threading.RateLimiting.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\System.Speech\Debug\net6.0\System.Speech.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\System.Speech\src\System.Speech.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\Microsoft.Extensions.DependencyInjection.Specification.Tests\Debug\net462\Microsoft.Extensions.DependencyInjection.Specification.Tests.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\Microsoft.Extensions.DependencyInjection.Specification.Tests\src\Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\Microsoft.Extensions.Hosting.Systemd\Debug\net6.0\Microsoft.Extensions.Hosting.Systemd.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\Microsoft.Extensions.Hosting.Systemd\src\Microsoft.Extensions.Hosting.Systemd.csproj]
D:\a\_work\1\s\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\a\_work\1\s\artifacts\bin\Microsoft.Extensions.Hosting.WindowsServices\Debug\net462\Microsoft.Extensions.Hosting.WindowsServices.xml' to be packed was not found on disk. [D:\a\_work\1\s\src\libraries\Microsoft.Extensions.Hosting.WindowsServices\src\Microsoft.Extensions.Hosting.WindowsServices.csproj]

```

I was able to repro after rebasing my branch to the latest bits. Not sure why I didn't see it before rebasing:
```
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\Microsoft.XmlSerializer.Generator\Release\netstandard2.0\dotnet-Microsoft.XmlSerializer.Generator.xml' to be packed was not found on disk. [D:\runtime\src\libraries\Microsoft.XmlSerializer.Generator\src\Microsoft.XmlSerializer.Generator.csproj]
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\Microsoft.Bcl.AsyncInterfaces\Release\net462\Microsoft.Bcl.AsyncInterfaces.xml' to be packed was not found on disk. [D:\runtime\src\libraries\Microsoft.Bcl.AsyncInterfaces\src\Microsoft.Bcl.AsyncInterfaces.csproj]
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\System.Runtime.Serialization.Schema\Release\net7.0\System.Runtime.Serialization.Schema.xml' to be packed was not found on disk. [D:\runtime\src\libraries\System.Runtime.Serialization.Schema\src\System.Runtime.Serialization.Schema.csproj]
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\System.Threading.RateLimiting\Release\net462\System.Threading.RateLimiting.xml' to be packed was not found on disk. [D:\runtime\src\libraries\System.Threading.RateLimiting\src\System.Threading.RateLimiting.csproj]
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\Microsoft.Bcl.Cryptography\Release\net462\Microsoft.Bcl.Cryptography.xml' to be packed was not found on disk. [D:\runtime\src\libraries\Microsoft.Bcl.Cryptography\src\Microsoft.Bcl.Cryptography.csproj]
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\System.Numerics.Tensors\Release\net462\System.Numerics.Tensors.xml' to be packed was not found on disk. [D:\runtime\src\libraries\System.Numerics.Tensors\src\System.Numerics.Tensors.csproj]
D:\runtime\.dotnet\sdk\8.0.100-preview.1.23115.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'D:\runtime\artifacts\bin\System.Speech\Release\net6.0\System.Speech.xml' to be packed was not found on disk. [D:\runtime\src\libraries\System.Speech\src\System.Speech.csproj]
```

--- a/eng/intellisense.targets
+++ b/eng/intellisense.targets
@@ -5,22 +5,20 @@
   </PropertyGroup>
 
   <Target Name="VerifyAssemblySupportsDocsXmlGeneration"
-    Condition="'$(UseIntellisensePackageDocXmlFile)' != 'true'">
-    <Error
-      Condition="'$(IsPartialFacadeAssembly)' == 'true'"
-      Text="The 'UseIntellisensePackageDocXmlFile' property is not supported for partial facade assemblies: $(AssemblyName)" />
-    <Error
-      Condition="'$(GeneratePlatformNotSupportedAssemblyMessage)' != ''"
-      Text="The 'UseIntellisensePackageDocXmlFile' property is not supported for assemblies that throw PlatformNotSupportedException: $(AssemblyName)" />
+          Condition="'$(UseIntellisensePackageDocXmlFile)' != 'true'">
+    <Error Text="The 'UseIntellisensePackageDocXmlFile' property is not supported for partial facade assemblies: $(AssemblyName)"
+           Condition="'$(IsPartialFacadeAssembly)' == 'true'" />
+    <Error Text="The 'UseIntellisensePackageDocXmlFile' property is not supported for assemblies that throw PlatformNotSupportedException: $(AssemblyName)"
+           Condition="'$(GeneratePlatformNotSupportedAssemblyMessage)' != ''" />
   </Target>
 
-  <PropertyGroup>
-    <NoWarn Condition="'$(UseIntellisensePackageDocXmlFile)' == 'true'">$(NoWarn);1591</NoWarn>
+  <PropertyGroup Condition="'$(UseIntellisensePackageDocXmlFile)' == 'true'">
+    <NoWarn>$(NoWarn);1591</NoWarn>
     <IntellisensePackageXmlRootFolder>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'microsoft.private.intellisense', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles'))</IntellisensePackageXmlRootFolder>
     <IntellisensePackageXmlFilePathFromNetFolder>$([MSBuild]::NormalizePath('$(IntellisensePackageXmlRootFolder)', 'net', '1033', '$(AssemblyName).xml'))</IntellisensePackageXmlFilePathFromNetFolder>
     <IntellisensePackageXmlFilePathFromDotNetPlatExtFolder>$([MSBuild]::NormalizePath('$(IntellisensePackageXmlRootFolder)', 'dotnet-plat-ext', '1033', '$(AssemblyName).xml'))</IntellisensePackageXmlFilePathFromDotNetPlatExtFolder>
-    <IntellisensePackageXmlFilePath Condition="'$(UseIntellisensePackageDocXmlFile)' == 'true' and Exists($(IntellisensePackageXmlFilePathFromNetFolder))">$(IntellisensePackageXmlFilePathFromNetFolder)</IntellisensePackageXmlFilePath>
-    <IntellisensePackageXmlFilePath Condition="'$(IntellisensePackageXmlFilePath)' == '' and '$(UseIntellisensePackageDocXmlFile)' == 'true' and Exists($(IntellisensePackageXmlFilePathFromDotNetPlatExtFolder))">$(IntellisensePackageXmlFilePathFromDotNetPlatExtFolder)</IntellisensePackageXmlFilePath>
+    <IntellisensePackageXmlFilePath Condition="'$(IntellisensePackageXmlFilePath)' == '' and Exists($(IntellisensePackageXmlFilePathFromNetFolder))">$(IntellisensePackageXmlFilePathFromNetFolder)</IntellisensePackageXmlFilePath>
+    <IntellisensePackageXmlFilePath Condition="'$(IntellisensePackageXmlFilePath)' == '' and Exists($(IntellisensePackageXmlFilePathFromDotNetPlatExtFolder))">$(IntellisensePackageXmlFilePathFromDotNetPlatExtFolder)</IntellisensePackageXmlFilePath>
   </PropertyGroup>
 
   <ItemGroup>
@@ -31,7 +29,7 @@
   <!-- Replace the default xml file generated in the obj folder with the one that comes from the docs feed. -->
   <Target Name="ChangeDocumentationFileForPackaging"
           BeforeTargets="CopyFilesToOutputDirectory;DocumentationProjectOutputGroup"
-          Condition="'$(UseIntellisensePackageDocXmlFile)' == 'true'">
+          Condition="'$(IntellisensePackageXmlFilePath)' != ''">
     <ItemGroup>
       <DocFileItem Remove="@(DocFileItem)" />
       <DocFileItem Include="$(IntellisensePackageXmlFilePath)" />

